{
 "errors": {
  "docker.api": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/__init__.py'",
    ""
   ]
  },
  "docker.api.build": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/build.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/build.py'",
    ""
   ]
  },
  "docker.api.client": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/client.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/client.py'",
    ""
   ]
  },
  "docker.api.config": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/config.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/config.py'",
    ""
   ]
  },
  "docker.api.container": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/container.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/container.py'",
    ""
   ]
  },
  "docker.api.daemon": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/daemon.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/daemon.py'",
    ""
   ]
  },
  "docker.api.exec_api": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/exec_api.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/exec_api.py'",
    ""
   ]
  },
  "docker.api.image": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/image.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/image.py'",
    ""
   ]
  },
  "docker.api.network": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/network.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/network.py'",
    ""
   ]
  },
  "docker.api.plugin": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/plugin.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/plugin.py'",
    ""
   ]
  },
  "docker.api.secret": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/secret.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/secret.py'",
    ""
   ]
  },
  "docker.api.service": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/service.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/service.py'",
    ""
   ]
  },
  "docker.api.swarm": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/swarm.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/swarm.py'",
    ""
   ]
  },
  "docker.api.volume": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/volume.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/api/volume.py'",
    ""
   ]
  },
  "docker.context": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/__init__.py'",
    ""
   ]
  },
  "docker.context.api": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/api.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/api.py'",
    ""
   ]
  },
  "docker.context.config": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/config.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/config.py'",
    ""
   ]
  },
  "docker.context.context": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/context.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/context/context.py'",
    ""
   ]
  },
  "docker.credentials": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/__init__.py'",
    ""
   ]
  },
  "docker.credentials.constants": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/constants.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/constants.py'",
    ""
   ]
  },
  "docker.credentials.errors": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/errors.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/errors.py'",
    ""
   ]
  },
  "docker.credentials.store": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/store.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/store.py'",
    ""
   ]
  },
  "docker.credentials.utils": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/utils.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/credentials/utils.py'",
    ""
   ]
  },
  "docker.models": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/models/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/models/__init__.py'",
    ""
   ]
  },
  "docker.transport": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/__init__.py'",
    ""
   ]
  },
  "docker.transport.basehttpadapter": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/basehttpadapter.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/basehttpadapter.py'",
    ""
   ]
  },
  "docker.transport.npipeconn": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/npipeconn.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/npipeconn.py'",
    ""
   ]
  },
  "docker.transport.npipesocket": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/npipesocket.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/npipesocket.py'",
    ""
   ]
  },
  "docker.transport.sshconn": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/sshconn.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/sshconn.py'",
    ""
   ]
  },
  "docker.transport.ssladapter": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/ssladapter.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/ssladapter.py'",
    ""
   ]
  },
  "docker.transport.unixconn": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/unixconn.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/transport/unixconn.py'",
    ""
   ]
  },
  "docker.types": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/__init__.py'",
    ""
   ]
  },
  "docker.types.base": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/base.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/base.py'",
    ""
   ]
  },
  "docker.types.containers": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/containers.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/containers.py'",
    ""
   ]
  },
  "docker.types.daemon": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/daemon.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/daemon.py'",
    ""
   ]
  },
  "docker.types.healthcheck": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/healthcheck.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/healthcheck.py'",
    ""
   ]
  },
  "docker.types.networks": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/networks.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/networks.py'",
    ""
   ]
  },
  "docker.types.services": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/services.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/services.py'",
    ""
   ]
  },
  "docker.types.swarm": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/swarm.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/types/swarm.py'",
    ""
   ]
  },
  "docker.utils": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/__init__.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/__init__.py'",
    ""
   ]
  },
  "docker.utils.build": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/build.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/build.py'",
    ""
   ]
  },
  "docker.utils.config": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/config.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/config.py'",
    ""
   ]
  },
  "docker.utils.decorators": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/decorators.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/decorators.py'",
    ""
   ]
  },
  "docker.utils.fnmatch": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/fnmatch.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/fnmatch.py'",
    ""
   ]
  },
  "docker.utils.json_stream": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/json_stream.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/json_stream.py'",
    ""
   ]
  },
  "docker.utils.ports": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/ports.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/ports.py'",
    ""
   ]
  },
  "docker.utils.proxy": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/proxy.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/proxy.py'",
    ""
   ]
  },
  "docker.utils.socket": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/socket.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/socket.py'",
    ""
   ]
  },
  "docker.utils.utils": {
   "exception": "[Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/utils.py'",
   "traceback": [
    "Traceback (most recent call last):",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/symbol_exporter/symbol_inspection.py\", line 36, in single_file_extraction",
    "    path=file_name, project=jedi.Project(\"\".join(top_dir))",
    "  File \"/usr/share/miniconda/envs/test/lib/python3.7/site-packages/jedi/api/__init__.py\", line 111, in __init__",
    "    with open(path, 'rb') as f:",
    "FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpn92puqle/lib/python3.6/site-packages/docker/utils/utils.py'",
    ""
   ]
  }
 },
 "symbols": [
  "docker.APIClient",
  "docker.Context",
  "docker.DockerClient",
  "docker.TLSConfig",
  "docker.__title__",
  "docker.__version__",
  "docker.api",
  "docker.api.APIClient",
  "docker.auth",
  "docker.auth.AuthConfig",
  "docker.auth.INDEX_NAME",
  "docker.auth.INDEX_URL",
  "docker.auth.TOKEN_USERNAME",
  "docker.auth._load_legacy_config",
  "docker.auth.convert_to_hostname",
  "docker.auth.decode_auth",
  "docker.auth.encode_header",
  "docker.auth.get_config_header",
  "docker.auth.get_credential_store",
  "docker.auth.load_config",
  "docker.auth.log",
  "docker.auth.parse_auth",
  "docker.auth.resolve_authconfig",
  "docker.auth.resolve_index_name",
  "docker.auth.resolve_repository_name",
  "docker.auth.split_repo_name",
  "docker.client",
  "docker.client.DockerClient",
  "docker.client.from_env",
  "docker.constants",
  "docker.constants.BYTE_UNITS",
  "docker.constants.CONTAINER_LIMITS_KEYS",
  "docker.constants.DEFAULT_DATA_CHUNK_SIZE",
  "docker.constants.DEFAULT_DOCKER_API_VERSION",
  "docker.constants.DEFAULT_HTTP_HOST",
  "docker.constants.DEFAULT_MAX_POOL_SIZE",
  "docker.constants.DEFAULT_NPIPE",
  "docker.constants.DEFAULT_NUM_POOLS",
  "docker.constants.DEFAULT_NUM_POOLS_SSH",
  "docker.constants.DEFAULT_SWARM_ADDR_POOL",
  "docker.constants.DEFAULT_SWARM_SUBNET_SIZE",
  "docker.constants.DEFAULT_TIMEOUT_SECONDS",
  "docker.constants.DEFAULT_UNIX_SOCKET",
  "docker.constants.DEFAULT_USER_AGENT",
  "docker.constants.INSECURE_REGISTRY_DEPRECATION_WARNING",
  "docker.constants.IS_WINDOWS_PLATFORM",
  "docker.constants.MINIMUM_DOCKER_API_VERSION",
  "docker.constants.STREAM_HEADER_SIZE_BYTES",
  "docker.constants.WINDOWS_LONGPATH_PREFIX",
  "docker.context",
  "docker.credentials",
  "docker.errors",
  "docker.errors.APIError",
  "docker.errors.BuildError",
  "docker.errors.ContainerError",
  "docker.errors.ContextAlreadyExists",
  "docker.errors.ContextException",
  "docker.errors.ContextNotFound",
  "docker.errors.DeprecatedMethod",
  "docker.errors.DockerException",
  "docker.errors.ImageLoadError",
  "docker.errors.ImageNotFound",
  "docker.errors.InvalidArgument",
  "docker.errors.InvalidConfigFile",
  "docker.errors.InvalidRepository",
  "docker.errors.InvalidVersion",
  "docker.errors.MissingContextParameter",
  "docker.errors.NotFound",
  "docker.errors.NullResource",
  "docker.errors.StreamParseError",
  "docker.errors.TLSParameterError",
  "docker.errors.create_api_error_from_http_exception",
  "docker.errors.create_unexpected_kwargs_error",
  "docker.from_env",
  "docker.models",
  "docker.models.configs.Config",
  "docker.models.configs.model",
  "docker.models.containers.Container",
  "docker.models.containers.ExecResult",
  "docker.models.containers.RUN_CREATE_KWARGS",
  "docker.models.containers.RUN_HOST_CONFIG_KWARGS",
  "docker.models.containers._create_container_args",
  "docker.models.containers._host_volume_from_bind",
  "docker.models.images.Image",
  "docker.models.images.RegistryData",
  "docker.models.images.normalize_platform",
  "docker.models.networks.Network",
  "docker.models.nodes.Node",
  "docker.models.nodes.model",
  "docker.models.plugins.Plugin",
  "docker.models.plugins.model",
  "docker.models.resource.Collection",
  "docker.models.resource.Collection.model",
  "docker.models.resource.Model",
  "docker.models.resource.Model.id_attribute",
  "docker.models.resource.Model.short_id",
  "docker.models.secrets.Secret",
  "docker.models.secrets.model",
  "docker.models.services.CONTAINER_SPEC_KWARGS",
  "docker.models.services.CREATE_SERVICE_KWARGS",
  "docker.models.services.ContainerSpec",
  "docker.models.services.InvalidArgument",
  "docker.models.services.PLACEMENT_KWARGS",
  "docker.models.services.Placement",
  "docker.models.services.Service",
  "docker.models.services.ServiceMode",
  "docker.models.services.TASK_TEMPLATE_KWARGS",
  "docker.models.services.TaskTemplate",
  "docker.models.services._get_create_service_kwargs",
  "docker.models.services.create_unexpected_kwargs_error",
  "docker.models.swarm.APIClient",
  "docker.models.swarm.APIError",
  "docker.models.swarm.Swarm",
  "docker.models.swarm.id_attribute",
  "docker.models.swarm.short_id",
  "docker.models.volumes.Volume",
  "docker.tls",
  "docker.tls.TLSConfig",
  "docker.transport",
  "docker.types",
  "docker.types.ContainerSpec",
  "docker.types.Placement",
  "docker.types.ServiceMode",
  "docker.types.TaskTemplate",
  "docker.utils",
  "docker.version",
  "docker.version.version",
  "docker.version.version_info",
  "docker.version_info"
 ]
}